### ISSUE TITLE
Profile switch on is not working!

### ISSUE THREAD

**Issue by carsten-h** [mammuth/ha-fritzbox-tools#97]
Hello!

I installed the tools via integration pane.
I got a few profile switches, which I can switch off. But it is not possible to switch them on again. The switch is showing that they are on, but nothing changes inside the Fritz!Box.

I am using the standard:
![Bildschirmfoto 2020-05-22 um 23 53 03](https://user-images.githubusercontent.com/44360828/82712043-68bdab00-9c87-11ea-8c29-5248fcf3614e.png)

Also I am getting this messages very often:
`Update of switch.fritzbox_profile_xxxxxxx is taking over 10 seconds
Update of switch.fritzbox_profile_yyyyyyy is taking over 10 seconds
Update of switch.fritzbox_profile_zzzzzzz is taking over 10 seconds
Update of switch.fritzbox_profile_wwwwwww is taking over 10 seconds`

This is happening in HA 0.110.1 but it also happened in 0.109.6.

**Comment by mammuth** [comment]
Is this the same issue as in https://github.com/mammuth/ha-fritzbox-tools/issues/90? Or does it also happen, when eg 1 minute elapsed between turning it off and on ?

**Comment by carsten-h** [comment]
In this case I wanted to switch on one hour after switching off.

**Comment by mammuth** [comment]
Okay, could you set your logging level to debug and paste the fritzbox_tools related logs on pastebin?
To do so, add the following to your `configuration.yaml` (or adapt your logger declaration accordingly):
```yaml
logger:
  default: error
  logs:
    custom_components.fritzbox_tools: debug
```

**Comment by carsten-h** [comment]
Yes, I can. I switched "WDMyCloud" off at 10:03:xx and back on at 10:13:xx.
[fritztools.log](https://github.com/mammuth/ha-fritzbox-tools/files/4671417/fritztools.log)

It is not so big, so I pasted it here.

**Comment by carsten-h** [comment]
Is there something more I can help you? Or is this information enough?

**Comment by mammuth** [comment]
I didn't have the time to debug the issue yet.

**Comment by carsten-h** [comment]
No problem, I have the time to wait!

**Comment by mammuth** [comment]
Probably closed by #110

Released as pre-release version 1.11.0, feel free to test @carsten-h
